<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MillMainWindow</class>
 <widget class="QMainWindow" name="MillMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MillMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7,1">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Fehér: lehelyezhető korongok:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="remainingWhite">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>megszerzett korongok: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="takenWhite">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="tableFrame">
      <property name="baseSize">
       <size>
        <width>7</width>
        <height>7</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" rowminimumheight="0" columnminimumwidth="0">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Fekete: lehelyezhető korongok:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="remainingBlack">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>megszerzett korongok:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="takenBlack">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
